### PR TITLE
added treatment page with content and links

### PR DESCRIPTION
**Pull request by jduds27**

### CODE REVIEW

File: src/components/treatment/treatment.component.jsx [hunk 1]
```diff
@@ -1,14 +1,151 @@
...
+        <div className='primary-container'>
+          <h3 className='primary-container-header'>
+            Treatment
+          </h3>
+          <h1 className='primary-container-header'>
+            Exploring treatment options for Excessive Sleepiness in OSA
+          </h1>
+          <div className='image-content-container'>
+            <div className='image-container'>
+              <img className='image-small' src={doctorSmall} alt='Exploring treatment options for excessive sleepiness in OSA' />
```
Comment (rphovley) [inline src/components/treatment/treatment.component.jsx]: Image should be full width on mobile

File: src/components/treatment/treatment.component.jsx [hunk 2]
```diff
@@ -1,14 +1,151 @@
...
+            <div className='content-info'>
+              <p>
+                The first-line treatment of choice for OSA is typically continuous positive airway pressure (CPAP); however, ES may persist despite CPAP adherence.<sup className='super'>1-3</sup>
+              </p>
+              <p>
+                Clinical tools are available to help HCPs proactively monitor ES in OSA. ATS Guidelines recommend routinely asking patients with OSA about their sleepiness and educating patients and families about risks associated with ES.<sup className='super'>4</sup> Tools for assessing ES include self-reported measures such as the Epworth Sleepiness Scale (ESS) and the Functional Outcomes of Sleep Questionnaire (FOSQ)/FOSQ-10, which evaluates the effect of ES on the patient’s quality of life.<sup className='super'>5,6</sup>
+              </p>
+            </div>
+          </div>
+          <h2 className='chart-header'>
```
Comment (rphovley) [inline src/components/treatment/treatment.component.jsx]: H3 instead of H2

File: src/components/treatment/treatment.component.jsx [hunk 3]
```diff
@@ -1,14 +1,151 @@
...
+            If you find your OSA patients have unresolved ES, consider these options to help manage their ES.
+          </h2>
+        </div>
+        <div className='grey-card-wrapper'>
+          <div className='grey-card'>
+            <div className='primary-container'>
+              <img src={moon} className='grey-card-image' alt='Treatment objectives for excessive sleepiness in OSA icon' />
+              <h2 className='chart-header'>
```
Comment (rphovley) [inline src/components/treatment/treatment.component.jsx]: H3 instead of H2

File: src/components/treatment/treatment.component.jsx [hunk 4]
```diff
@@ -1,14 +1,151 @@
...
+              </h2>
+              <div className='content-info'>
+                Ensure patients are getting an adequate amount of sleep and that their primary airway therapy is optimized and working properly.<sup className='super'>7</sup>
+              </div>
+            </div>
+          </div>
+          <div className='grey-card'>
+            <div className='primary-container'>
+              <img src={medicine} className='grey-card-image' alt='Pharmacotherapyy treatment options for excessive sleepiness in OSA icon' />
+              <h2 className='chart-header'>
```
Comment (rphovley) [inline src/components/treatment/treatment.component.jsx]: H3 instead of H2

File: src/components/treatment/treatment.component.jsx [hunk 5]
```diff
@@ -1,14 +1,151 @@
...
+        </div>
+        <div className='primary-container'>
+          <div className='video-container'>
+            <VideoContainer video={Videos.management} />
+          </div>
+          <div className="grey-callout">
+            <img src={Icon} alt="Excessive sleepiness (ES) in OSA - video play icon" />
+            <div className="grey-callout-title">Are your patients underreporting their sleepiness?</div>
+            <div className="grey-callout-content">Consider these tools for monitoring ES in OSA.</div>
+            <SiteLink className="grey-callout-link" to="Video Library">View Now</SiteLink>
```
Comment (rphovley) [inline src/components/treatment/treatment.component.jsx]: The annotated pdf says this is supposed to link to the "Monitoring" page?

Comment (jduds27) [reply src/components/treatment/treatment.component.jsx]: fixed

File: src/components/treatment/treatment.component.jsx (hunk 4 above)
Comment (jduds27) [reply src/components/treatment/treatment.component.jsx]: fixed

File: src/components/treatment/treatment.component.jsx (hunk 3 above)
Comment (jduds27) [reply src/components/treatment/treatment.component.jsx]: fixed

File: src/components/treatment/treatment.component.jsx (hunk 2 above)
Comment (jduds27) [reply src/components/treatment/treatment.component.jsx]: fixed

File: src/components/treatment/treatment.component.jsx (hunk 1 above)
Comment (jduds27) [reply src/components/treatment/treatment.component.jsx]: fixed

**Pull request merged**